main.go

```diff
@@ -132,6 +131,7 @@ func initFlags(fs *pflag.FlagSet) {
 
 func validateFlags() error {
 	switch options.ProviderIDFormatType(options.ProviderIDFormat) {
+	// Deprecated: ProviderIDFormatV1 is deprecated and will be removed in a future release.
 	case options.ProviderIDFormatV1:
 		setupLog.Info("Using v1 version of ProviderID format")
```

Comment:
Lets update this log line to mention whatever you added in comments. So the user will get to know about the deprecation.

Reply:
Sure will do it.

Reply:
Done.